tests/deal.II/grid_tools_halo_layer_01.output

```diff
@@ -0,0 +1,3303 @@
+0.00000 0.250000 1.00000 2 1
+0.00000 0.250000 0.750000 2 1
+
+0.00000 0.500000 0.750000 2 1
+0.250000 0.500000 0.750000 2 1
+0.250000 0.500000 1.00000 2 1
+0.00000 0.500000 1.00000 2 1
+0.00000 0.500000 0.750000 2 1
+
+0.00000 0.250000 0.750000 2 1
+0.00000 0.500000 0.750000 2 1
+
+0.250000 0.250000 0.750000 2 1
+0.250000 0.500000 0.750000 2 1
+
+0.250000 0.250000 1.00000 2 1
+0.250000 0.500000 1.00000 2 1
+
+0.00000 0.250000 1.00000 2 1
+0.00000 0.500000 1.00000 2 1
+
+0.250000 0.250000 0.750000 2 1
+0.500000 0.250000 0.750000 2 1
+0.500000 0.250000 1.00000 2 1
+0.250000 0.250000 1.00000 2 1
+0.250000 0.250000 0.750000 2 1
+
+0.250000 0.500000 0.750000 2 1
+0.500000 0.500000 0.750000 2 1
+0.500000 0.500000 1.00000 2 1
+0.250000 0.500000 1.00000 2 1
+0.250000 0.500000 0.750000 2 1
+
+0.250000 0.250000 0.750000 2 1
+0.250000 0.500000 0.750000 2 1
+
+0.500000 0.250000 0.750000 2 1
+0.500000 0.500000 0.750000 2 1
+
+0.500000 0.250000 1.00000 2 1
+0.500000 0.500000 1.00000 2 1
+
+0.250000 0.250000 1.00000 2 1
+0.250000 0.500000 1.00000 2 1
+
+0.500000 0.00000 0.500000 2 3
+0.750000 0.00000 0.500000 2 3
+0.750000 0.00000 0.750000 2 3
+0.500000 0.00000 0.750000 2 3
+0.500000 0.00000 0.500000 2 3
+
+0.500000 0.250000 0.500000 2 3
+0.750000 0.250000 0.500000 2 3
+0.750000 0.250000 0.750000 2 3
+0.500000 0.250000 0.750000 2 3
+0.500000 0.250000 0.500000 2 3
+
+0.500000 0.00000 0.500000 2 3
+0.500000 0.250000 0.500000 2 3
+
+0.750000 0.00000 0.500000 2 3
+0.750000 0.250000 0.500000 2 3
+
+0.750000 0.00000 0.750000 2 3
+0.750000 0.250000 0.750000 2 3
+
+0.500000 0.00000 0.750000 2 3
+0.500000 0.250000 0.750000 2 3
+
+0.750000 0.00000 0.500000 2 1
+1.00000 0.00000 0.500000 2 1
+1.00000 0.00000 0.750000 2 1
+0.750000 0.00000 0.750000 2 1
+0.750000 0.00000 0.500000 2 1
+
+0.750000 0.250000 0.500000 2 1
+1.00000 0.250000 0.500000 2 1
+1.00000 0.250000 0.750000 2 1
+0.750000 0.250000 0.750000 2 1
+0.750000 0.250000 0.500000 2 1
+
+0.750000 0.00000 0.500000 2 1
+0.750000 0.250000 0.500000 2 1
+
+1.00000 0.00000 0.500000 2 1
+1.00000 0.250000 0.500000 2 1
+
+1.00000 0.00000 0.750000 2 1
+1.00000 0.250000 0.750000 2 1
+
+0.750000 0.00000 0.750000 2 1
+0.750000 0.250000 0.750000 2 1
+
+0.500000 0.250000 0.500000 2 3
+0.750000 0.250000 0.500000 2 3
+0.750000 0.250000 0.750000 2 3
+0.500000 0.250000 0.750000 2 3
+0.500000 0.250000 0.500000 2 3
+
+0.500000 0.500000 0.500000 2 3
+0.750000 0.500000 0.500000 2 3
+0.750000 0.500000 0.750000 2 3
+0.500000 0.500000 0.750000 2 3
+0.500000 0.500000 0.500000 2 3
+
+0.500000 0.250000 0.500000 2 3
+0.500000 0.500000 0.500000 2 3
+
+0.750000 0.250000 0.500000 2 3
+0.750000 0.500000 0.500000 2 3
+
+0.750000 0.250000 0.750000 2 3
+0.750000 0.500000 0.750000 2 3
+
+0.500000 0.250000 0.750000 2 3
+0.500000 0.500000 0.750000 2 3
+
+0.750000 0.250000 0.500000 2 1
+1.00000 0.250000 0.500000 2 1
+1.00000 0.250000 0.750000 2 1
+0.750000 0.250000 0.750000 2 1
+0.750000 0.250000 0.500000 2 1
+
+0.750000 0.500000 0.500000 
```

Comment:
That's a lot of output. Could the output  made smaller,or the mesh be made coarser?